install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -285,8 +285,9 @@ type ProxyConfig struct {
 }
 
 type ConfigcatProxyConfig struct {
-	BaseUrl      string `json:"baseUrl"`
-	PollInterval string `json:"pollInterval"`
+	BaseUrl       string `json:"baseUrl"`
+	PollInterval  string `json:"pollInterval"`
+	FromConfigMap string `json:"fromConfigMap"`
```

Comment:
I'm trying to understand: why do we sometimes use it as a `string` and sometimes as a `bool`?

Reply:
Talked about this with Pudong. This is because when used as a string, it indicates the name of the ConfigMap and in other places it is only used to indicate whether one should be used.